charts/stable/wyoming-whisper/questions.yaml

```diff
@@ -232,27 +234,68 @@ questions:
             required: true
             default: tiny-int8
             enum:
-              - value: base
-                description: base
+              - value: custom
+                description: custom
+              - value: tiny-int8
+                description: tiny-int8
+              - value: tiny
+                description: tiny
+              - value: tiny.en
+                description: tiny.en
               - value: base-int8
                 description: base-int8
-              - value: medium-int8
-                description: medium-int8
-              - value: small
-                description: small
+              - value: base
+                description: base
+              - value: base.en
+                description: base.en
               - value: small-int8
                 description: small-int8
-              - value: tiny
-                description: tiny
-              - value: tiny-int8
-                description: tiny-int8
+              - value: distil-small.en
+                description: distil-small.en
+              - value: small
+                description: small
+              - value: small.en
+                description: small.en
+              - value: medium-int8
+                description: medium-int8
+              - value: distil-medium.en
+                description: distil-medium.en
+              - value: medium
+                description: medium
+              - value: medium.en
+                description: medium.en
+              - value: large
+                description: large
+              - value: large-v1
+                description: large-v1
+              - value: distil-large-v2
+                description: distil-large-v2
+              - value: large-v2
+                description: large-v2
+              - value: large-v3
+                description: large-v3
+        - variable: custom_model
+          label: Custom Model
+          description: 'Path to a converted model directory, or a CTranslate2-converted Whisper model ID from the HuggingFace Hub like "Systran/faster-distil-whisper-small.en".'
+          schema:
+            type: string
+            required: false
```

Comment:
```suggestion
```

Reply:
Oh I forgot to ask about this, if I set required here to true (which it should be because if you set model to custom and don't specify what the custom model should be, the app won't start), but `show_if` is not satisfied and this required option is not actually visible in App config, will this prevent the user from saving the config?
I'm just not sure if TrueNAS will handle this properly. Ideally it should ignore this "required" variable if it's not actually shown, but if it can't handle edge cases like this, it could still want to set to something even if the user is unable to.

Reply:
Will it work if instead of `false` I set the same condition `[[model, "=", "custom"]]` as with `show_if`?